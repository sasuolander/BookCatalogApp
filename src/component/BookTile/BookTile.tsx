import React, { FunctionComponent } from "react";

type BookTileProps = {
  id: Number;
};
export const BookTile: FunctionComponent<BookTileProps> = ({ id }) => {
  return <div>"books number `{id}`"</div>;
};
