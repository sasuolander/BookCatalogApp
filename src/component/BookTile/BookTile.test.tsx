import React from 'react'
import {BookTile} from './BookTile'
import { mount, shallow, render } from "enzyme";

it("render component book tile",()=>{
    const component = shallow(<BookTile/>)
    expect(component).toMatchSnapshot();
})