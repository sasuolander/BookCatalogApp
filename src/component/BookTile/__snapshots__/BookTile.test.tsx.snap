// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Book tile render component book tile 1`] = `
<div>
  "books number \`
  \`"
</div>
`;

exports[`render component book tile 1`] = `
<div>
  "books number \`
  \`"
</div>
`;
